### ISSUE TITLE
Split Maintenance Date into Individual Rows in MMC Preventative Maintenance Private Dataset `mudg-bik3`

### ISSUE THREAD

**Issue by KaroEngstrom** [cityofaustin/atd-data-tech#13235]
From Allyson R: 

For[ this feed](https://datahub.austintexas.gov/Transportation-and-Mobility/Preventative-Maintenance/mudg-bik3/explore/query/SELECT%0A%20%20%60signal_id%60%2C%0A%20%20%60location_name%60%2C%0A%20%20%60signal_type%60%2C%0A%20%20%60turn_on_date%60%2C%0A%20%20%60signal_pm_max_fiscal_year%60%2C%0A%20%20%60pm_completed_date%60%2C%0A%20%20%60maintenance_date%60%2C%0A%20%20%60owner%60%2C%0A%20%20%60modified_date%60%2C%0A%20%20%60id%60/page/filter), we are wanting to split the maintenance date, which is a string of date, into individual rows. The dates are hard to parse out. 

@chiaberry  I think @johnclary  just worked on this or related dataset, possible handled this field in that one. 

Looping in @ChristinaTremel for AMD PM and communications.

**Comment by ChristinaTremel** [comment]
This was a need when the preventative maintenance records were stored on the historical object and the newer work order preventative maintenance object. We originally had the signals object as the API source and then added the connected preventative maintenance fields from both objects so the dates were showing up as a string. But now that we've migrated all the records into the work orders preventative maintenance object and are sourcing the API table from that, there should only be one date per row!

Closing this!
